src/command.rs

```diff
@@ -5,6 +5,7 @@ pub(crate) enum Command {
   Operation(Operation),
   Repl,
   Resize { rows: usize, cols: usize },
+  Print,
```

Comment:
Super small nitpick: keep the enum order the same as they appear in the match statement (i.e sorted), but other than that this looks good to me!